src/ethereum_test_fixtures/schemas/blockchain/headers.py

```diff
@@ -0,0 +1,190 @@
+"""
+Define genesisHeader schema for filled .json tests
+"""
+
+from typing import Any, Optional
+
+from pydantic import BaseModel, Field, model_validator
+from rlp import decode as rlp_decode
```

Comment:
To fix the mypy issues, we should be able to extend [`./stubs/`](https://github.com/ethereum/execution-spec-tests/tree/2d361623337f300d36b067f99c418873cec7b4fa/stubs) folder with the functions used here. 

`rlp` - https://pypi.org/project/rlp/ doesn't appear to support typing yet, ethereum/pyrlp#143

Reply:
I added one, check if it is how it supposed to be. 
but still it does not detect in vscode where rlp is supposed to be imported from.

but strangely all works